drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -69,7 +69,7 @@ def self.apply_fragment(item)
     def self.seed_record_form_definitions
       record_forms.each do |identifier, form_definition|
         role = identifier.upcase.to_sym
-        raise "unrecognized role #{role}" unless Hmis::Form::Definition::FORM_ROLES.key?(role)
+        next unless Hmis::Form::Definition::FORM_ROLES.key?(role)
```

Comment:
On line 98 you check the same thing but with `role.to_sym` I'm guessing they both should be `to_sym` maybe?

Reply:
Its already a symbol in the other place